prow-jobs/pingcap-tidb-latest-presubmits.yaml

```diff
@@ -44,7 +44,7 @@ presubmits:
     - name: pingcap/tidb/ghpr_unit_test
       agent: jenkins
       decorate: false # need add this.
-      always_run: true
+      skip_if_only_changed: "(\\.md|Dockerfile|OWNERS|OWNERS_ALIASES)$"
```

Comment:
The current pipelines will automatically trigger after a new commit push. Will this behavior change after adjustment?

Reply:
no, it judge the the diff of the whole pull request, not the commit changed scope.

Reply:
My concern is whether the pipeline can be automatically triggered based on the new commit after modifying the code and pushing it.